demos/artifactory/README.md

```diff
@@ -18,6 +20,22 @@ unclassified:
           password: "${ARTIFACTORY_PASSWORD}"
 ```
 
+Since 3.11.0
+
+```yaml
+unclassified:
+  artifactorybuilder:
+    useCredentialsPlugin: true
+    jfrogInstances:
+      - instanceId: artifactory
+        url: http://acme.com/artifactory
+        deployerCredentialsConfig:
+          credentialsId: "artifactory"
+        resolverCredentialsConfig:
+          username: artifactory_user
+          password: "${ARTIFACTORY_PASSWORD}"
+```
+
```

Comment:
Move this above this section above the before tbh. We should always promote newer vesions.